locustempus/templates/main/course_create.html

```diff
@@ -29,8 +29,17 @@
             {% endif %}
         </div>
         <div class="form-group">
-            <label for="id_description">Description:</label>  {# not hooked up #}
-            <textarea name="description" cols="40" rows="3" class="form-control" maxlength="200" placeholder="A brief summary of your workspace. NOT HOOKED UP YET" id="id_description"></textarea>
+            <label for="{{ form.description.id_for_label }}">Description:</label>
+            {% if form.description.errors %}
+                {{ form.description|add_class:"form-control is-invalid"|attr:"data-cy:workspace-description" }}
+                <div class="invalid-feedback">
+                    {% for error in form.description.errors %}
+                        {{ error }}
+                    {% endfor %}
+                </div>
+            {% else %}
+                <textarea name="id_description" cols="40" rows="3" class="form-control" maxlength="200" placeholder="A brief summary of your workspace." id="id_description" data-cy="field-workspace-description"></textarea>
```

Comment:
The name field here should be `description` ... this is why the CourseForm is not picking up the description.